tests/Unit/ConfigMgrCBDsc.ResourceHelper.tests.ps1

```diff
@@ -765,4 +765,76 @@ InModuleScope $script:subModuleName {
             }
         }
     }
+
+    Describe "$moduleResourceName\ConvertTo-CimCMScheduleString" {
+        $mockCimRefreshScheduleDay = (New-CimInstance -ClassName DSC_TestCimInstance `
+            -Namespace root/microsoft/Windows/DesiredStateConfiguration `
+            -Property @{
+                'RecurInterval' = 'Days'
+                'RecurCount'    = 6
+            } -ClientOnly
+        )
+        $mockCimRefreshScheduleHours = (New-CimInstance -ClassName DSC_TestCimInstance `
+            -Namespace root/microsoft/Windows/DesiredStateConfiguration `
+            -Property @{
+                'RecurInterval' = 'Hours'
+                'RecurCount'    = 7
+            } -ClientOnly
+        )
+        $mockCimRefreshScheduleMin = (New-CimInstance -ClassName DSC_TestCimInstance `
+            -Namespace root/microsoft/Windows/DesiredStateConfiguration `
+            -Property @{
+                'RecurInterval' = 'Minutes'
+                'RecurCount'    = 50
+            } -ClientOnly
+        )
+        $scheduleConvertDays = @{
+            DayDuration    = 0
+            DaySpan        = 6
+            HourDuration   = 0
+            HourSpan       = 0
+            MinuteDuration = 0
+            MinuteSpan     = 0
+        }
+        $scheduleConvertHours = @{
+            DayDuration    = 0
+            DaySpan        = 0
+            HourDuration   = 0
+            HourSpan       = 7
+            MinuteDuration = 0
+            MinuteSpan     = 0
+        }
+        $scheduleConvertMin = @{
+            DayDuration    = 0
+            DaySpan        = 0
+            HourDuration   = 0
+            HourSpan       = 0
+            MinuteDuration = 0
+            MinuteSpan     = 50
+        }
+        $cimInputParam = @{
+            CimClassName   = 'DSC_TestCimInstance'
+            ScheduleString = '0001200000100038'
+        }
+        Context 'When return is as expected' {
+            It 'Should return desired result for hearbeat discovery hours' {
```

Comment:
The It block says the same thing for all three. Is this directly related to discovery hours? 

Also, I know it works by using the same input for since the function is being mocked but it would be nice to have the different inputs for testing that reflect the outputs being mocked.

Reply:
Changed the descriptions. Will address the rest soon.

Reply:
I gave it different inputs, but since you said it may be redundant, I didn't do any assert mocks

Reply:
It may be good practice to do the Assert since we are saying that mock should be ran in all the scenarios. I would verify with Jeff his thoughts.

Reply:
I spoke to Jeff and got this switched over.